vck/src/commonMain/kotlin/at/asitplus/wallet/lib/jws/JwsService.kt

```diff
@@ -102,7 +106,8 @@ interface VerifierJwsService {
 
 class DefaultJwsService(private val cryptoService: CryptoService) : JwsService {
 
-    override val algorithm: JwsAlgorithm = cryptoService.keyPairAdapter.signingAlgorithm.toJwsAlgorithm().getOrThrow()
+    override val algorithm: JwsAlgorithm =
+        cryptoService.keyMaterial.signatureAlgorithm.toJwsAlgorithm().getOrThrow()
 
     // TODO: Get from crypto service
```

Comment:
I know that TODO isn't new, but can it be resolved now?

Reply:
not yet, as we don't yet support encryption in a multiplatform manner. this should then be fixed, once we do